src/index.ts

```diff
@@ -1,10 +1,13 @@
 export { hemi } from "./chains/hemi.js"
 export { hemiSepolia } from "./chains/hemi-sepolia.js"
 
-export { hemiPublicOpNodeActions } from "./decorators/public-op-node-actions.js"
 export { getBtcFinalityByBlockHash } from "./actions/get-btc-finality-by-block-hash.js"
 
 export { hemiPublicBitcoinKitActions } from "./decorators/public-bitcoin-kit-actions.js"
+export { hemiPublicBitcoinTunnelManagerActions } from "./decorators/public-bitcoin-tunnel-manager-actions.js"
+export { hemiPublicBitcoinVaultActions } from "./decorators/public-bitcoin-vault-actions.js"
```

Comment:
In the current version of the file we export the "actions" object and each individual action. See lines 4-5 and 7-8. Both are "from ./decorators...", then "from ./actions...".

I guess we should do the same with the tunnel and vault actions.

Reply:
Added exports here [6e44bd8](https://github.com/hemilabs/hemi-viem/pull/7/commits/6e44bd8ab45b6c870185309655edbd5365cedf11). I did not use `*` for decorators as I hope we never have more than one thing there (btw, we need now a decorator of decorators, so we don't need to create each file 😆 )